cmd/podman/pull.go

```diff
@@ -127,7 +126,7 @@ func pullCmd(c *cliconfig.PullValues) error {
 	} else {
 		authfile := getAuthFile(c.String("authfile"))
 		spec := image
-		systemContext := common.GetSystemContext("", authfile, false)
+		systemContext := image2.GetSystemContext("", authfile, false)
```

Comment:
Not due to this review, but I think we can drop image2 now, I think the other include ending in image has gone away.  (Or maybe I've been staring at the screen too long today and missed it).

Reply:
image2 is used because they have a variable named image in use.